json-server/db.js

```diff
@@ -627,8 +627,8 @@ module.exports = {
       recurringType: 'weekly',
       dates: [
         {
-          startTime: '2017-08-06T16:30:00.000Z',
```

Comment:
I believe your issue is that the creation date goes past after some time : create a new Date instead of changing the month :)